### ISSUE TITLE
Yamui Listview

### ISSUE THREAD

**Issue by n3R0d0T** [jcaillon/YamuiFramework#2]
It would be great if this has a listview.

**Comment by jcaillon** [comment]
Hi, the latest version has a sort of listview (not grid only list), i can also display trees : 

![image](https://user-images.githubusercontent.com/11553075/36152767-65475104-10cc-11e8-9a31-7be5f32ef623.png)

I must update this repo with the latest version and give sample usage of the listview but i don't have the time at the moment ;/

https://github.com/jcaillon/3P/tree/master/YamuiFramework

The list components are there if you are interested : 
https://github.com/jcaillon/3P/tree/master/YamuiFramework/Controls/YamuiList

**Comment by n3R0d0T** [comment]
i have to display my data like this in picture.
![caplist](https://user-images.githubusercontent.com/8027932/36153395-105d631c-1109-11e8-93e5-d9d13b7be3eb.PNG)

How should i do it in yamuilist? thanks in advance

**Comment by jcaillon** [comment]
Yep, you can't use yamuilist for that, it is not designed for "gridview" (i.e. with several columns).
I suggest to use this amazing project : 
http://objectlistview.sourceforge.net/cs/index.html

**Comment by n3R0d0T** [comment]
Thanks for your reply. I would like to try making yamuilistview. I'm really a fan of this framework.

**Comment by jcaillon** [comment]
Glad you like it. Do not hesitate to make a pull request if you manage to implement a grid view :)
